adyenofficial.php

```diff
@@ -944,6 +948,31 @@ public function displayForm()
             'hint' => $this->l('')
         );
 
+        if ($this->versionChecker->isPrestaShop16()) {
+            $fields_form[1]['form']['input'][] = array(
+                'type' => 'radio',
+                'label' => $this->l('Remove collapsable payment display'),
+                'name' => 'ADYEN_REMOVE_PAYMENT_DISPLAY_COLLAPSE',
+                'values' => array(
```

Comment:
is there a way to have a default value?

Reply:
Based on http://doc.prestashop.com/display/PS16/Using+the+HelperForm+class#UsingtheHelperFormclass-Radiobutton and my tests, there doesn't seem to be a way of specifying a default value

Reply:
thanks for checking it! 👍